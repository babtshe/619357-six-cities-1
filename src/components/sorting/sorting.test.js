import React from 'react';
import renderer from 'react-test-renderer';
import Sorting from './sorting';

it(`Sorting should render correctly`, ()=>{
  const mockOptions = [{
    name: `test`
  }];
  const tree = renderer
  .create(
      <Sorting
        isOpened = {true}
        options = {mockOptions}
      />)
  .toJSON();

  expect(tree).toMatchSnapshot();
});
