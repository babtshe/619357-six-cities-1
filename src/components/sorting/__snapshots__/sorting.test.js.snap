// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sorting should render correctly 1`] = `
<form
  action="#"
  className="places__sorting"
  method="get"
>
  <span
    className="places__sorting-caption"
  >
    Sort by 
  </span>
  <span
    className="places__sorting-type"
    onClick={[Function]}
    tabIndex="0"
  >
    <svg
      className="places__sorting-arrow"
      height="4"
      width="7"
    >
      <use
        xlinkHref="#icon-arrow-select"
      />
    </svg>
  </span>
  <ul
    className="places__options places__options--custom places__options--opened"
  >
    <li
      className="places__option"
      onClick={[Function]}
      tabIndex="0"
    >
      test
    </li>
  </ul>
</form>
`;
